goth/runner/container/__init__.py

```diff
@@ -97,6 +97,13 @@ class DockerContainer:
     function.
     """
 
+    restart: Callable
```

Comment:
`mypy` complained when checking `probe.py` that this attribute/method is not present.